src/client/components/EventsPage/EventsPage.component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { StudyGroupLogo } from '../StudyGroupLogo/StudyGroupLogo';
+import './EventsPage.styles.css';
+
+function EventsPage() {
+  return (
+    <div>
+      <div>Study Groups for Class15</div>
+      <div className="main-container">
+        <div className="top-part">
+          <div className="logo-part">
+            <div className="event-logo">
+              <StudyGroupLogo />
+            </div>
+            <div className="right-part">
+              <div className="event-group">Elastic - Week 46</div>
+              <div className="event-date">Wednesday | 11 Nov 20 | 10:00</div>
+              <div className="mentor-name">mentors name</div>
+              <div className="event-link">link to registoration</div>
+            </div>
+          </div>
+        </div>
+
+        <button type="button" className="register-btn">
```

Comment:
Since `Register yourself` button is actually part of each card, you should handle the `onClick` and pass it through the props. But! You also need to indicate _which one_ was clicked if there are multiple events on this page.